### PR TITLE
Dg update

### PR DESCRIPTION
**Pull request by ReflectiveObsidian**

### CODE REVIEW

File: docs/diagrams/PersonClassDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+package "Attribute classes"{
+abstract Class Attribute
+Class StringAttribute
+Class IntegerAttribute
+Class DateAttribute
+Class BirthdayAttribute
+Class NameAttribute
+Class PhoneNumberAttribute
+Class SexAttribute
```
Comment (Bandov) [inline docs/diagrams/PersonClassDiagram.puml:27]: yep thats how the attributes work with the person object

File: docs/diagrams/UiClassDiagram.puml [hunk 2]
```diff
@@ -30,31 +31,39 @@ HiddenOutside ..> Ui
...
+DisplaySection ----|> UiPart
+AnyListSection -|> UiPart
+
+
+AllContactsSection ----|> UiPart
+PersonList ---|> UiPart
+
+PersonList .right.> Model
+UiManager -right> Logic
+MainWindow -right> Logic
```
Comment (Bandov) [inline docs/diagrams/UiClassDiagram.puml:66]: ok yep

File: docs/images/ModelClassDiagram.png
Comment (Bandov) [inline docs/images/ModelClassDiagram.png:1]: do we not need to show how the attributes interact iwth the model?

File: docs/images/RelationshipClassDiagram.png
Comment (Bandov) [inline docs/images/RelationshipClassDiagram.png:1]: LGTM

File: docs/images/UiClassDiagram.png
Comment (Bandov) [inline docs/images/UiClassDiagram.png:1]: LGTM

**Review by Bandov — APPROVED** [review]
LGTM

File: docs/images/ModelClassDiagram.png
Comment (ReflectiveObsidian) [reply docs/images/ModelClassDiagram.png:1]: will do

**Pull request merged**